app/controllers/simplified-account/settings/team-members/team-members.controller.test.js

```diff
@@ -0,0 +1,115 @@
+const proxyquire = require('proxyquire')
+const sinon = require('sinon')
+const { expect } = require('chai')
+const User = require('../../../../models/User.class')
+const userFixtures = require('../../../../../test/fixtures/user.fixtures')
+
+const ACCOUNT_TYPE = 'test'
+const SERVICE_ID = 'service-id-123abc'
+
+let req, res, responseStub, getServiceUsersStub, getInvitedUsersStub, teamMembersController
+
+const getController = (stubs = {}) => {
+  return proxyquire('./team-members.controller', {
+    '../../../../utils/response': { response: stubs.response },
+    '../../../../services/user.service':
+      { getServiceUsers: stubs.getServiceUsers, getInvitedUsers: stubs.getInvitedUsers }
+  })
+}
+
+const setupTest = (method, additionalReqProps = {}, additionalStubs = {}) => {
+  responseStub = sinon.spy()
+  const adminUser = new User(userFixtures.validUserResponse({
+    external_id: 'user-id-for-admin-user',
+    service_roles: {
+      service: {
+        service: { external_id: SERVICE_ID },
+        role: { name: 'admin' }
+      }
+    }
+  }))
+  const viewOnlyUser = new User(userFixtures.validUserResponse(
+    {
+      external_id: 'user-id-for-view-only-user',
+      service_roles: {
+        service:
+            {
+              service: { external_id: SERVICE_ID },
+              role: { name: 'view-only' }
+            }
+      }
+    }))
+  const users = [adminUser, viewOnlyUser]
+  getServiceUsersStub = sinon.stub().resolves(users)
+
+  const invitedAdminUser = { email: 'invited-admin-user@user.gov.uk', role: 'admin' }
+  const invitedViewOnlyUser = { email: 'invited-view-only-user@user.gov.uk', role: 'view-only' }
+  const invitedUsers = [invitedAdminUser, invitedViewOnlyUser]
+  getInvitedUsersStub = sinon.stub().resolves(invitedUsers)
+
+  teamMembersController = getController({
+    response: responseStub,
+    getServiceUsers: getServiceUsersStub,
+    getInvitedUsers: getInvitedUsersStub,
+    ...additionalStubs
+  })
+  res = {
+    redirect: sinon.spy()
+  }
+  req = {
+    user: new User({
+      service_roles: [
+        {
+          role: {
+            name: 'admin'
+          },
+          service: {
+            external_id: SERVICE_ID
+          }
+        }
+      ]
+    }),
```

Comment:
could this be simplified to use the fixture you've already created above?
```suggestion
    user: adminUser,
```

Reply:
good point, done.